doc/design.md

```diff
@@ -232,18 +280,18 @@ SelectParameter <|-- BaseParameter
 Covers:
 * [`req~parameter-types~1`](system_requirements.md#validation-of-parameter-values)
 
-Needs: impl, utest, itest
+Needs: impl, utest
 
 #### Conditional Parameters
 `dsn~conditional-parameters~1`
 
 Conditions for conditional parameters are represented by JSON structures, see [design decision](#alternative-options-to-represent-conditional-parameters) against alternative options to represent conditional parameters.
 
+**This is not yet implemented**
```

Comment:
OFT has a feature to mark requirements a draft. Then you don't have to cover them with bogus coverage.